integration/asciidoc/src/main/java/org/archifacts/integration/asciidoc/TableDocElement.java

```diff
@@ -65,14 +65,21 @@ private void renderStartEndTag(final StringBuilder stringBuilder) {
 	}
 
 	private void renderHeader(final StringBuilder stringBuilder) {
-		for (final TableDocElementColumn<T> column : columns) {
-			stringBuilder.append('|');
-			if (column.title != null) {
-				stringBuilder.append(column.title);
+		final boolean atLeastOneColumnTitleIsNotNull = columns
+				.stream()
+				.map(c -> c.title)
```

Comment:
In TableDocElementTest you introduced getters in order to use method refeences. I would prefer to use this approach here as well --> TableDocElementColumn::getTitle

Reply:
I added further (private) getters to access the fields.